### ISSUE TITLE
Create Datadog Dashboards for Max CFI Metrics

### ISSUE THREAD

**Issue by amylai-nava** [department-of-veterans-affairs/abd-vro#1993]
The Employee Experience team wants to monitor max CFI MVP and max CFI API metrics post launch.

## To do

- [x] For max CFI MVP metrics: work with Steve Albers to set up a dashboard in VA.gov Datadog
- [ ] For max CFI API metrics: work with VRO to set up a dashboard for EE's deployed application logs on LHDI Datadog

**Relevant tickets/conversations:**
- https://github.com/department-of-veterans-affairs/abd-vro/issues/1911
- [Slack thread](https://dsva.slack.com/archives/C01BVF2A3V0/p1692823272669479?thread_ts=1692138742.709369&cid=C01BVF2A3V0) about our decision to calculate the number of “educated” users who didn’t choose tinnitus by subtracting the ones who did from the total number of “educated” users
- [VRO slack thread](https://dsva.slack.com/archives/C01Q7979Z7D/p1693341950117069) requesting their support to create a dashboard in LHDI Datadog

**Comment by dfitchett** [comment]
Blocked by https://github.com/department-of-veterans-affairs/abd-vro/issues/1992

**Comment by amylai-nava** [comment]
@dfitchett I'm going to consider this done and create a new ticket to track the LHDI Datadog dashboard.
